cli/src/commands/init.js

```diff
@@ -160,7 +160,7 @@ const handler = async ({ force, name, cwd, lib }) => {
         reporter.info('Installing @dhis2/cli-app-scripts...')
         await exec({
             cmd: 'yarn',
-            args: ['add', '--dev', '@dhis2/cli-app-scripts'],
+            args: ['add', '--dev', '@dhis2/cli-app-scripts@alpha'],
```

Comment:
this should be updated to remove alpha?

Reply:
Indeed 🙂 https://github.com/dhis2/app-platform/pull/896 

(I think I'll just consider this an approval, and go ahead and merge that 😁 )